engine/execution/computation/execution_verification_test.go

```diff
@@ -161,7 +161,7 @@ func Test_ExecutionMatchesVerification(t *testing.T) {
 		err = testutil.SignTransaction(addKeyTx, accountAddress, accountPrivKey, 0)
 		require.NoError(t, err)
 
-		minimumStorage, err := cadence.NewUFix64("0.00008021")
+		minimumStorage, err := cadence.NewUFix64("0.00008164")
```

Comment:
The storage for accounts has inflated a bit. This is good to keep in mind! We will need to run the storage sync migration.